Source/WebKit/UIProcess/API/Cocoa/WKWebsiteDataRecord.mm

```diff
@@ -42,6 +42,7 @@
 NSString * const WKWebsiteDataTypeWebSQLDatabases = @"WKWebsiteDataTypeWebSQLDatabases";
 NSString * const WKWebsiteDataTypeIndexedDBDatabases = @"WKWebsiteDataTypeIndexedDBDatabases";
 NSString * const WKWebsiteDataTypeServiceWorkerRegistrations = @"WKWebsiteDataTypeServiceWorkerRegistrations";
+NSString * const WKWebsiteDataTypeFileSystem = @"WKWebsiteDataTypeFileSystem";
```

Comment:
I think you want to make _WKWebsiteDataTypeFileSystem start using this string definition too (instead of "_WKWebsiteDataTypeFileSystem"), to keep the effect of the old client code working during the (admittedly small) transition period. This a more minor issue -- just a UI effect rather than a crash. But nice to keep everything smooth.

Reply:
Sure, updated.